### PR TITLE
Subscription Database Documentation

### PR DESCRIPTION
**Pull request by micylt**
Added more documentation to sub_to and sub_from modules.

Let me know if you would like to see anything else included...

Thanks.

### CODE REVIEW

File: blockchain/db/postgres/sub_from_db.py [hunk 1]
```diff
@@ -22,6 +22,19 @@
 language governing permissions and limitations under the Apache License.
 """
 
+"""
+                ***** server-side subscription dao ******
+this dao is used for storing and retrieving server-side subscription data.
+when a server receives new subscription data from a client, the data is stored.
```
Comment (Voxelot) [inline blockchain/db/postgres/sub_from_db.py]: I'd maybe reword this to simply:
> Stores subscription info from subscribers (client nodes) for the data provider.

File: blockchain/db/postgres/sub_to_db.py [hunk 2]
```diff
@@ -22,6 +22,19 @@
 language governing permissions and limitations under the Apache License.
 """
 
+"""
+                ***** client-side subscription dao ******
+this dao is used for storing and retrieving client-side subscription data.
+when a client receives a new subscription transaction, the subscription data is stored.
```
Comment (Voxelot) [inline blockchain/db/postgres/sub_to_db.py]: Maybe this?
> Stores subscription info for the subscriber node when a subscription transaction is received.

**Comment by micylt** [comment]
@Voxelot I like that. Made the adjustments.
